source/native-plugins/_data.all.cpp

```diff
@@ -83,6 +83,22 @@ static const NativePluginDescriptor sNativePluginDescriptors[] = {
     /* copyright */ "GNU GPL v2+",
     DESCFUNCS
 },
+{
+    /* category  */ NATIVE_PLUGIN_CATEGORY_UTILITY,
+    /* hints     */ NATIVE_PLUGIN_IS_RTSAFE,
+    /* supports  */ NATIVE_PLUGIN_SUPPORTS_EVERYTHING,
+    /* audioIns  */ 0,
+    /* audioOuts */ 0,
+    /* midiIns   */ 1,
+    /* midiOuts  */ 2,
+    /* paramIns  */ 0,
+    /* paramOuts */ 0,
+    /* name      */ "MIDI Channel A/B",
+    /* label     */ "midichanab",
+    /* maker     */ "falkTX",
```

Comment:
You have maker here as me, but the actual plugin code has it as "Milk Brewster".
Which one is it?
If you want to set yourself as the author, that is completely fine, but please change in all places on the PR and Update the header comments too regarding it.
No need for "Copyright (C) 2012-2015 Filipe Coelho", just use your usual one-liner for these things.
I assume you're okay with GPL v2+ ?

Reply:
Thanks for catching that. Unless I'm mistaken, it's corrected below.

I've left your copyright header in midi-channel-ab.c also, given it's based on your code. Should I for sure be adding a line to files other than midi-channel-ab.c? I don't mind either way.

I'm fine with GPL v2+